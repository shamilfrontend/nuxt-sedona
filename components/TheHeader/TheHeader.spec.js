import { shallowMount } from '@vue/test-utils'
import Component from './index'

describe('TheHeader', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Component, {
      stubs: ['nuxt-link']
    })
  })

  test('mounts properly', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
