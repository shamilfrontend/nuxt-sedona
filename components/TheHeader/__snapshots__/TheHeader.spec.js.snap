// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheHeader should match snapshot 1`] = `
<header class="page-header">
  <nav class="main-menu">
    <nuxt-link-stub to="/" tabindex="1"><img src="./img/logo.png" alt="Седона" class="logo"></nuxt-link-stub>
    <ul class="main-menu__list">
      <li class="main-menu__item">
        <nuxt-link-stub to="/info" class="main-menu__link">
          Информация
        </nuxt-link-stub>
      </li>
      <li class="main-menu__item">
        <nuxt-link-stub to="/media" class="main-menu__link">
          Фото и видео
        </nuxt-link-stub>
      </li>
      <li class="main-menu__item">
        <nuxt-link-stub to="/map" class="main-menu__link">
          Карта штата
        </nuxt-link-stub>
      </li>
      <li class="main-menu__item">
        <nuxt-link-stub to="/hotels" class="main-menu__link">
          Гостиницы
        </nuxt-link-stub>
      </li>
    </ul>
  </nav>
</header>
`;
